### PR TITLE
dxScheduler – add TestCafe tests for DragAndDrop functionality (day view)

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by jsdmitry — DISMISSED** [review]

**Review by AntonSermyazhko — DISMISSED** [review]

**Review by andrewmakarov — DISMISSED** [review]

File: testing/functional/helpers/scheduler.test.helper.ts [hunk 1]
```diff
@@ -11,6 +11,30 @@ export default class SchedulerTestHelper {
         return this.scheduler.find(`.dx-scheduler-appointment`).nth(index);
     }
 
+    getAppointmentByTitle(title) {
```
Comment (Olexi) [inline testing/functional/helpers/scheduler.test.helper.ts]: I would suggest adding the second parameter `index`, which would be in `nth` statement

File: testing/functional/tests/scheduler/dragAndDrop/presets/widget.setup.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        resources: [{
+            fieldExpr: "resourceId",
+            dataSource: schedulerResources,
+            label: "Color"
+        }],
+        views: [mode],
+        currentView: mode,
+        currentDate: new Date(2018, 3, 20),
+        startDayHour: 9,
+        useColorAsDefault: true
```
Comment (Olexi) [inline testing/functional/tests/scheduler/dragAndDrop/presets/widget.setup.ts]: Why you use this parameter here? Read https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/resources/#useColorAsDefault

**Review by Olexi — CHANGES_REQUESTED** [review]

File: testing/functional/tests/scheduler/dragAndDrop/presets/widget.setup.ts (hunk 2 above)
Comment (ghost) [reply testing/functional/tests/scheduler/dragAndDrop/presets/widget.setup.ts]: Yes, it's not needed. I will fix it next time.

File: testing/functional/helpers/scheduler.test.helper.ts (hunk 1 above)
Comment (ghost) [reply testing/functional/helpers/scheduler.test.helper.ts]: Yes. This is a very good point.

**Review by Olexi — APPROVED** [review]

**Pull request merged**